website/src/pages/profile.js

```diff
@@ -294,11 +297,6 @@ function DaoProfile ({ daoAddress, history }) {
   }
 
   const { organisation } = data
-  // if the organisation itself is null, it means
-  // it was not found in the DB
-  if (!organisation) {
-    return <ProfileNotFound history={history} />
-  }
```

Comment:
How do empty profiles work now?

Reply:
This was actually redundant; this case is handled further up: https://github.com/1Hive/apiary/blob/master/website/src/pages/profile.js#L80